cloudbuild.yaml

```diff
@@ -227,14 +227,26 @@ steps:
     - GO111MODULE=on
 
 #
-# Run the e2e tests
+# Run the e2e tests with FeatureGates
 #
 
 - name: 'e2e-runner'
+  args: ['PlayerTracking=true&ContainerPortAllocation=true']
   waitFor:
     - push-images
     - build-e2e
 
+#
+# Run the e2e tests without FeatureGates
+#
+
+- name: 'e2e-runner-release'
+  args: ['']
+  waitFor:
+    - e2e-runner
```

Comment:
This line could be redundany, as `consul lock` should not allow this to happen simultaneously.